python/python/glide/async_commands/server_modules/json.py

```diff
@@ -320,6 +320,47 @@ async def forget(
     )
 
 
+async def numincrby(
+    client: TGlideClient,
+    key: TEncodable,
+    path: TEncodable,
+    number: Union[int, float],
+) -> Optional[bytes]:
+    """
+    Increments or decrements the number of the JSON values at the specified `path' by `number` within the JSON document stored at `key`.
+
+    Args:
+        client (TGlideClient): The client to execute the command.
+        key (TEncodable): The key of the JSON document.
+        path (TEncodable): The path within the JSON document.
+        number (Union[int, float]): The number to increment or decrement by.
+
+    Returns:
+        Optional[bytes]:
+            For JSONPath ('path' starts with '$'):
+                Returns a bytes string representation of an array of bulk strings that indicates the new values after incrementing for each matched 'path'.
+                If a value is not a number, its corresponding return value will be 'None'.
```

Comment:
```suggestion
                If a value is not a number, its corresponding return value will be `null`.
```

remember its inside the bytes string, its not a python None